backend/src/kitconcept/intranet/distributions/intranet/content/content/195d96ca9952493cbffa69cda040b8b3/data.json

```diff
@@ -27,7 +27,7 @@
   "modified": "2025-07-25T15:25:07+00:00",
   "nav_title": null,
   "preview_caption_link": null,
-  "remoteUrl": "/features/content-types",
+  "remoteUrl": "http://nohost/Plone/features/content-types",
```

Comment:
remove this http://nohost/Plone

Reply:
How this got added here , not sure but removed it

Reply:
yes, it is a bug in plone.exportimport.